geoplanar/gap.py

```diff
@@ -224,13 +224,21 @@ def snap(geometry, threshold=0.5):
         pairs_to_snap = nearby_not_overlap[~duplicated]
     
         new_geoms = []
+        previous_geom=0
+        snapped_geom=0
```

Comment:
```suggestion
        previous_geom=None
        snapped_geom=None
```

I know it won't make a huge difference but it si easier to understand that the variable is not meant to be an integer.